src/Core/Checkout/Document/Zugferd/ZugferdBuilder.php

```diff
@@ -91,4 +106,21 @@ protected function matchByType(ZugferdDocument $document, OrderLineItemEntity $l
 
         $this->eventDispatcher->dispatch(new ZugferdInvoiceItemAddedEvent($document, $lineItem, $parentPosition), 'zugferd-item-added.' . $lineItem->getType());
     }
+
+    private function addPaymentInfo(ZugferdDocument $document, DocumentConfiguration $config, PaymentMethodEntity $paymentMethod): void
+    {
+        if ($paymentMethod->getTechnicalName() === 'payment_cashpayment') {
+            $document->getBuilder()->addDocumentPaymentMean(
+                typeCode: (string) ZugferdPaymentMeans::UNTDID_4461_10->value,
+                information: $paymentMethod->getName()
+            );
+        } elseif ($paymentMethod->getTechnicalName() === 'payment_invoicepayment' || $paymentMethod->getTechnicalName() === 'payment_prepayment') {
```

Comment:
This looks like you likely wanna extend this if your extension provides a new payment method, e.g. what about PayPal? Are we fine with only supporting these methods for the time being with no easy way for extension? 🤔